src/modules/clear.js

```diff
@@ -0,0 +1,9 @@
+const clearAll = document.querySelector('.clear-btn');
+
+const cleartodos = (taskstore) => {
+  taskstore.tasks = taskstore.tasks.filter((task) => task.completed === false);
+  taskstore.updateIndex();
```

Comment:
- [ ] For now, when we proceed with the actions to check the tasks checkbox, nothing happens. the action `completed` is not updated in your local storage to false or true, It remains `false` for every action. Kindly make sure that you fix this issue so that you can proceed with the clear all completed tasks action.

Reply:
- [ ] About your linters issues I don't know especially if it will fix your problem but you can try this, It worked sometimes for me:

Delete manually directory` npm` and `npm-cache` in ...AppData/Roaming in your user disc for your PC
Run in your terminal `npm cache clean --force`
Or try to delete the node_modules directory and the package-lock.json file from your project and the run `npm install` again.

Reply:
done 
i have it done on my code 
kindly check the local and see the interactivity of the checkbox
![required](https://user-images.githubusercontent.com/78634841/173194550-1b6616fc-c9a8-4d0e-bea9-dda76978c154.jpg)


kindly find attached the image above to see the response of the checkbox to complete true or false
if the checkbox is clicked the activity will change from complete false to complete true
kindly check this out 
Thanks